api/cronhook/start-cron.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# start-cron.sh
+
+printenv >> /etc/environment
```

Comment:
As you already said in https://github.com/desec-io/desec-stack/pull/38#issue-220883537, this writes out sensitive data. Can you justify why it is needed?